lib/Dialect/FIRRTL/Transforms/LowerClasses.cpp

```diff
@@ -437,6 +457,32 @@ static void populateTypeConverter(TypeConverter &converter) {
         return om::ListType::get(elementType);
       });
 
+  auto convertMapType = [&converter](auto type) -> std::optional<mlir::Type> {
```

Comment:
Good call re:lambda, should do same for the list conversions too probably (not in this PR, something I can touchup) :+1: .